.vscode-test.mjs

```diff
@@ -30,6 +31,19 @@ const configs = testSuiteFolderNames.map((folderName) => ({
   },
 }));
 
-const config = defineConfig(configs);
+const config = defineConfig({
+  tests: configs,
+  coverage: {
+    exclude: ['src/test/**', '**/node_modules/**', '**/dist/**'],
+    thresholds: {
+      global: {
+        statements: 80,
+        branches: 80,
+        functions: 80,
+        lines: 80,
+      },
+    },
```

Comment:
I'm not a code coverage expert, but I play one on TV.

The vscode-test cli doesn't provide the code coverage tooling, mocha provides general code coverage with the ability for other libraries to expand/extend on them.

The thresholds are from when I was trying to get the [nyc[(https://github.com/istanbuljs/nyc) tool to work, but the return on the amount of time required to get it working was not beneficial. I opted to keep the builtin coverage, but neglected to remove the thresholds object. If we were to use another library, options like these would fail the build if they were not met